build/jenkins_e2e.sh

```diff
@@ -31,8 +31,11 @@ if [[ ! -z "$(git diff --name-only pages)" ]]; then
   exit 1
 fi
 
-make
-go build -tags test github.com/google/cadvisor/integration/runner
+# Build & test with go 1.7
+docker run --rm \
```

Comment:
I'm not sure how to do that without cloning every make rule. Is that worth it? Locally I can control my go version, but with jenkins this is the recommended way.